apps/wallet-mobile/src/YoroiApp.tsx

```diff
@@ -38,18 +38,16 @@ setLogLevel(CONFIG.LOG_LEVEL)
 if (Boolean(Config.DISABLE_LOGBOX)) LogBox.ignoreAllLogs()
 
 const queryClient = new QueryClient()
-const amplitudeClient = getMetricsFactory(features.analytics ? 'amplitude' : 'mock')({
-  apiKey: Config.AMPLITUDE_API_KEY ?? '',
-})
-const metricsStorage = makeMetricsStorage()
+
+const metricsManager = features.analytics ? makeMetricsManager() : mockMetricsManager()
```

Comment:
feature flag will go away eventually -> module for now is behind it, only the mock version is injected.